index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-First</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="bg">
+    <header class="toolbar">
+        <img src="./assets/images/My Logo.png" alt="Logo">
+        <nav>
+        <a href="#">
+        <img src="./assets/images/dropdown.png" alt="Dropdown menu">
+        </a>
+        </nav>
+    </header>
+
+    <section class="headline">
+            <h1 class="name">I'm Haider. <br/>Glad to see you!</h1>
+
+            <p class="intro">
+                I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t hesitate to
+                contact me.
+            </p>
```

Comment:
- [ ] For the headline text, It's already great but I believe you could apply more weight and font size to make it as close as shown on the figma design. 
- [ ] Also, regarding the background image, I believe it could use a little arrangement so that it can be identical to the Figma design. Just some alignments. 
### Your design 
![image](https://user-images.githubusercontent.com/69077061/234977763-0c638f99-5283-4923-8b75-d15197b69461.png)
### Figma design 
![image](https://user-images.githubusercontent.com/69077061/234977990-3f789e19-05a2-4988-804e-8eabc0af8451.png)